data-load/user-load.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Script to load a batch of new users into OAuth for new service access.
+#
+# Parameters:
+#       1. ENV - [t3|t2|preprod|prod]
+#       2. CLIENT - the client Id and secret, comma-separated
```

Comment:
should be colon separated I think

Reply:
Good spot. Wrong words though does the right thing.